[{"id":554151760,"node_id":"R_kgDOIQevUA","name":"CVE-2020-0688-Python3","full_name":"TheKickPuncher/CVE-2020-0688-Python3","private":false,"owner":{"login":"TheKickPuncher","id":71475034,"node_id":"MDQ6VXNlcjcxNDc1MDM0","avatar_url":"https://avatars.githubusercontent.com/u/71475034?v=4","url":"https://api.github.com/users/TheKickPuncher","html_url":"https://github.com/TheKickPuncher","type":"User","site_admin":false},"html_url":"https://github.com/TheKickPuncher/CVE-2020-0688-Python3","description":"Exploit updated to use Python 3.","fork":false,"url":"https://api.github.com/repos/TheKickPuncher/CVE-2020-0688-Python3","created_at":"2022-10-19T10:36:10Z","updated_at":"2022-10-20T01:02:02Z","pushed_at":"2022-10-19T10:58:29Z","stargazers_count":1,"watchers_count":1,"forks_count":0,"archived":false,"disabled":false,"allow_forking":true,"is_template":false,"topics":["cve-2020-0688"],"visibility":"public","forks":0,"watchers":1,"default_branch":"main","score":1}]